docs: update status/metrics/peers endpoints + reference `rc.2.0` (#520)
hyperledger-iroha/iroha-2-docs: src/reference/torii-endpoints.md

#### Responses

| Code | Response                             | Description                                                                        |
| :--: | ------------------------------------ | ---------------------------------------------------------------------------------- |

> | 200  | OK, Transaction Accepted             | Transaction has been accepted, but is not yet guaranteed to have passed consensus. |

Reviewer: ```suggestion
| 200  | OK, Transaction Accepted             | The Transaction has been accepted, but is not yet guaranteed to have passed consensus. |
```

> | 400  | Bad Request, Transaction Rejected    | Transaction is rejected due to being malformed.                                    |

Reviewer: ```suggestion
| 400  | Bad Request, Transaction Rejected    | The Transaction has been rejected due to being malformed.                                    |
```

> | 500  | Internal Error, Transaction Rejected | Iroha could not accept transaction due to e.g. its queue being full.               |

Reviewer: ```suggestion
| 500  | Internal Error, Transaction Rejected | Iroha could not accept the transaction due to e.g. its queue being full.               |
```
